<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_alm_temp</class>
 <widget class="QDialog" name="Dlg_alm_temp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>体温报警限设置</string>
  </property>
  <widget class="QComboBox" name="m_cmbb_temp">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_l">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>80</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>报警低限</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>报警高限</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_ok">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>报警</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_h">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>90</number>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_cancel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>体温通道1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>体温通道2</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_h_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>90</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spb_l_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>80</number>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_temp_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
